preview_envs.go

```diff
@@ -36,3 +47,20 @@ func (pe PreviewEnvs) Create(stackId int64, env *CreatePreviewEnvInput) (*types.
 	}
 	return &updatedEnv, nil
 }
+
+// Update - PUT/PATCH /orgs/:orgName/stacks/:stack_id/preview_envs/:id
+func (pe PreviewEnvs) Update(stackId, envId int64, env *UpdatePreviewEnvInput) (*types.Environment, error) {
+	rawPayload, _ := json.Marshal(env)
+	res, err := pe.Client.Do(http.MethodPut, pe.envPath(stackId, envId), nil, nil, json.RawMessage(rawPayload))
+	if err != nil {
+		return nil, err
+	}
+
+	var updatedEnv types.Environment
+	if err := response.ReadJson(res, &updatedEnv); response.IsNotFoundError(err) {
```

Comment:
You can return ReadJsonPtr here instead of this entire block.

Reply:
Done, I keep forgetting about this method. I was a victim of copying from the method above here. I went ahead and updated both methods in this file.